README.md

```diff
@@ -3,14 +3,9 @@
 *Updated and maintained done by Ishaan Desai*
 
 ## 1. How to build the Fluent-preCICE adapter:
-  * Adapt `lnamd64/2ddp_host/user.udf` line 1 "CSOURCES=...": There are several main udf files
-  * The variable SOURCES needs one main udf file and the correspoding .c files:
-    + fsi_udf.c: This is the main ANSYS readable file used by Fluent to call functions during each iteration
-                 for FSI simulations. This ANSYS UDF file needs fsi.c and fsi.h for execution
```

Comment:
I do not have to change anything here. Therefore, I removed this line. Or is there anything that has to be added?

Reply:
These lines are there as an information to the user as to where the adapter source code is inserted and seen by ANSYS. if in the future there are more files (for e.g. `cht.c` for a CHT application), then the `"CSOURCES=..."` will need to be modified accordingly. 
I would suggest to keep these lines for the moment.